starlette/background.py

```diff
@@ -1,12 +1,16 @@
 import asyncio
 import typing
 
+import typing_extensions
```

Comment:
`typing_extensions` is only available for versions below Python 3.8 right now. I guess we'd want to change the setup.py?

Reply:
Agree, fixed

Reply:
But now it needs to verify the version, otherwise it will raise `ModuleNotFoundError` on 3.10.

Reply:
Agree, fixed again)